package.json

```diff
@@ -3,9 +3,11 @@
   "version": "1.3.1",
   "description": "SQL injection protection module",
   "main": "./SQL.js",
+  "types": "./SQL.d.ts",
   "scripts": {
     "test": "tap *.test.js",
     "test:security": "node ./sqlmap/sqlmap.js",
+    "test:typescript": "tsd",
```

Comment:
We should refactor our tests so that `npm test` runs both the tap and tsd tests (so this is run on CI).